### DISCUSSION TITLE
Add an option to hide taskbar when fullscreen / maximized.

### DISCUSSION THREAD

**Discussion by DeviousChimp550** [runelite/runelite#18636] [Ideas]
The windows taskbar is still visible when the client is fullscreen, which is quite frustrating. This can be remedied by autohiding the taskbar through windows settings, but I don't want to have to hide my taskbar JUST for runelite. Having a checkbox for "fullscreen", that also hides the taskbar would be beneficial I believe.

**Comment by cdfisher** [comment]
Try "Fullscreen" on the Plugin Hub.

I can't speak to how well it works, but it has enough users that I would imagine it at least mostly does the job.
